roles/dns_encryption/templates/dnscrypt-proxy.toml.j2

```diff
@@ -446,7 +450,7 @@ cache_neg_max_ttl = 600
 
   [sources.'public-resolvers']
   urls = ['https://raw.githubusercontent.com/DNSCrypt/dnscrypt-resolvers/master/v2/public-resolvers.md', 'https://download.dnscrypt.info/resolvers-list/v2/public-resolvers.md']
-  cache_file = 'public-resolvers.md'
+  cache_file = '/run/dnscrypt-proxy/public-resolvers.md'
```

Comment:
Wouldn't it break *BSD?

Reply:
Yes I guess it would, I didn't think of that. Should I add code to prepend `/run/dnscrypt-proxy/` to the file name only in the case of Ubuntu?

Reply:
Can we use a relative path as before?

Reply:
I didn't see a way to tell `dnscrypt-proxy` to write these files somewhere other than `/etc/dnscrypt-proxy`, which it can't because it doesn't run as root.

Reply:
then, I think to add a task, and ensure the directory `/run/dnscrypt-proxy/` exists, would be enough

Reply:
or just to not make it complex and place the file in `/tmp`

Reply:
We could use `/tmp`, I just thought using one of the `systemd` created directories was more elegant. :smile: I probably should have chosen `/var/cache/dnscrypt-proxy` anyway.

It seems like the root of this problem is a bug in the PPA package, perhaps.

I'll change it to `/tmp`. I'll do a test deploy before I commit.

Reply:
Thanks!